pkg/apis/serving/v1alpha2/framework_onnx.go

```diff
@@ -77,7 +77,7 @@ func (s *ONNXSpec) Validate(config *InferenceServicesConfig) error {
 	if err != nil {
 		return err
 	}
-	if ext := path.Ext(uri.Path); ext != ONNXFileExt || ext != "" {
+	if ext := path.Ext(uri.Path); ext != ONNXFileExt && ext != "" {
```

Comment:
do we need the same fix in v1beta1?

Reply:
I haven't yet ported the ONNX spec change to v1beta1 yet, wanted to merge this first, perhaps this is better off addressing first?

Reply:
sure, can you help rebase the master?

Reply:
sure, just pushed a rebase